src/components/PropertyValuePair/stories/PropertyName.stories.tsx

```diff
@@ -16,3 +16,19 @@ export const Default = Template.bind({});
 Default.args = {
     children: loremIpsum({ p: 1, avgSentencesPerParagraph: 1, avgWordsPerSentence: 4, random: false }).toString(),
 };
+
+export const EmphasisLabel = Template.bind({});
+EmphasisLabel.args = {
+    ...Default.args,
+    labelProps: {
+        emphasis: "strong",
+    },
+};
+
+export const RightIconabel = Template.bind({});
```

Comment:
Remove this story but add the example for additional elements directly to the default story of the Label component.